test/ibuf.c

```diff
@@ -34,17 +34,26 @@ ibuf_basic()
 
 	fail_unless(ibuf_used(&ibuf) == 0);
 
+	ptr = ibuf_alloc(&ibuf, UINT32_MAX);
+	fail_unless(ptr != NULL);
+	fail_unless(ibuf_used(&ibuf) == UINT32_MAX);
+
+	ibuf_reset(&ibuf);
+
+	fail_unless(ibuf_used(&ibuf) == 0);
+
 	footer();
 }
 
 int main()
 {
-	quota_init(&quota, UINT_MAX);
+	quota_init(&quota, QUOTA_MAX);
```

Comment:
The test fill fail on 32-bit platform. Probably not an issue, but please have @alyapunov to confirm.

Reply:
If we change unsigned to uint64_t and __builtin_clz to __builtin_clzll, there will be no need to test it.